assets/state-vfio-manager/0400_configmap.yaml

```diff
@@ -123,6 +125,44 @@ data:
         fi
     }
 
+    get_graphics_aux_dev() {
+        local gpu=$1
+        device_class_file=$(readlink -f "/sys/bus/pci/devices/$gpu/class")
+        device_class=$(cat "$device_class_file")
+        if [ "$device_class" == "0x030000" ]; then
+          if ls "/sys/bus/pci/devices/$gpu" | grep consumer >& /dev/null; then
+            aux_dev=$(ls "/sys/bus/pci/devices/$gpu" | grep consumer | awk -Fconsumer:pci: '{print $2}')
+            if [ "$device_class" == "" ]; then
+              echo "NONE"
+              return
+            fi
```

Comment:
Why are we checking for an empty device class string here? Couldn't we check this before checking the `consumer` link?

Reply:
You caught a copy/paste error. Meant to check for empty aux_dev.